server/config/regenerate-client.js

```diff
@@ -1,35 +0,0 @@
-#!/usr/bin/env node
-/***** WARNING: ES5 code only here. Not transpiled! *****/
-
-/**
- * Module dependencies/
- */
-var fs = require( 'fs' ),
-	path = require( 'path' ),
-	configPath = path.resolve( __dirname, '..', '..', 'config' ),
-	keysPath = path.resolve( configPath, 'client.json' ),
-	keys = JSON.parse( fs.readFileSync( keysPath, 'utf8' ) ),
-	config = require( './' ),
-	data = require( './parser' )( configPath, {
-		env: process.env.CALYPSO_ENV || 'development',
-		includeSecrets: false,
-		enabledFeatures: process.env.ENABLE_FEATURES,
-		disabledFeatures: process.env.DISABLE_FEATURES
-	} ),
-	obj = {};
-
-keys.forEach( function( key ) {
```

Comment:
@samouri I think we need to bring it back when we expose `clientData` in `server/config/parser.js`.